runtime/queries/elixir/injections.scm

```diff
@@ -23,3 +19,25 @@
       (string (quoted_content) @injection.content)
       (sigil (quoted_content) @injection.content)
   ])) (#set! injection.language "markdown"))
+
+; Phoenix Live View HEEx Sigils
+((sigil
+  (sigil_name) @_sigil_name
+  (quoted_content) @injection.content)
+ (#eq? @_sigil_name "H")
+ (#set! injection.language "heex")
+ (#set! injection.combined))
+
+; Phoenix Live View Component Macros
+(call 
+  (identifier) @_identifier
+  (arguments
+    (atom)+
+    (keywords (pair 
+      (keyword) 
```

Comment:
It looks like `:doc` is the only option that can take a string for now, but for future-proofing, should we also test that this `(keyword)` node is `#eq?` to "doc"?